<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(248, 248, 248);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>782</width>
      <height>419</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(248, 248, 248);</string>
    </property>
    <widget class="QWidget" name="login_page">
     <property name="styleSheet">
      <string notr="true">

#widget {
	background-color: rgb(255, 255, 255);
	border-radius:25px;
	border: 2px solid #55aa00;
}

#verticalLayout_3 {
	
	background-color: rgb(170, 0, 127);
}

QLabel {
	
	color: #767e89;
}

QLabel#KRITICAL_Label {
	color:  #38a752;
	font: 36pt &quot;Segoe UI&quot;;
}

QLabel#H_Label {
	color: rgb(0, 0, 127);
	font: 36pt &quot;Segoe UI&quot;;
}

QLabel#T_Label {
	color: rgb(0, 0, 127);
	font: 36pt &quot;Segoe UI&quot;;
}



QLabel#Welcome {
	background-color: rgb(255, 255, 255);
	color: #767e89;
}

QLabel#smallLabel {
	background-color: rgb(255, 255, 255);
	color: #767e89;
}

QLabel#smallLabel_2 {
	background-color: rgb(255, 255, 255);
	color: #767e89;
}

QLineEdit {
	background-color: rgb(248, 248, 248);
	border: none;
	border-bottom: 2px solid #cccccc;
	height: 20px;
}

QLineEdit:focus {
	border-bottom: 2px solid #38a752;
}

QPushButton {
	margin-top: 10px;
	background-color: rgb(85, 255, 127);
	border: 2px solid #55aa00;
	color: rgb(23, 107, 10);
	color: #fff;
	border-radius: 5px;
	padding: 6px;
}

QPushButton:hover, QPushButton:clicked {
	background-color: #38a752;

}</string>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>170</y>
        <width>241</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="1" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>45</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>44</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>11</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>11</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>20</number>
         </property>
         <item>
          <widget class="QLabel" name="Welcome">
           <property name="font">
            <font>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">horizontalAlignment: AlignHCenter;
color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>Welcome</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>15</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="spacing">
              <number>3</number>
             </property>
             <item>
              <widget class="QLabel" name="smallLabel">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Username</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="username_Edit">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>13</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>3</number>
             </property>
             <item>
              <widget class="QLabel" name="smallLabel_2">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Password</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="password_Edit">
               <property name="font">
                <font>
                 <pointsize>13</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="loginButton">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>Login</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="KRITICAL_Label">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>50</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>KRITICAL</string>
      </property>
     </widget>
     <widget class="QLabel" name="T_Label">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>100</y>
        <width>31</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>T</string>
      </property>
     </widget>
     <widget class="QLabel" name="H_Label">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>0</y>
        <width>31</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>H</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="main_menu_page">
     <property name="styleSheet">
      <string notr="true">
QLabel#KRITICAL_Label_2 {
	color:  #38a752;
	font: 36pt &quot;Segoe UI&quot;;
}

QLabel#H_Label_2 {
	color: rgb(0, 0, 127);
	font: 36pt &quot;Segoe UI&quot;;
}

QLabel#T_Label_2 {
	color: rgb(0, 0, 127);
	font: 36pt &quot;Segoe UI&quot;;
}

QPushButton{
	color: rgb(255, 255, 255);
	border-radius: 10px;
}

QPushButton#logOut_Button{
	
	color: rgb(255, 255, 255);
	background-color: rgb(148, 148, 148);
	font-size: 15px;
	setWordWrap: true;
}

QPushButton#logOut_Button:hover, QPushButton#logOut_Button:clicked {
	background-color: rgb(71, 71, 71);
}


QPushButton#newTemplate_Button{
	
	background-color: rgb(0, 143, 0);
	font-size: 20px;
	setWordWrap: true;
}

QPushButton#loadTemplate_Button{
	background-color: rgb(85, 85, 255);
	font-size: 20px;
	setWordWrap: true;
}

QPushButton#testMode_Button{
	
	background-color: rgb(255, 85, 0);
	font-size: 20px;
	setWordWrap: true;
}

</string>
     </property>
     <widget class="QLabel" name="H_Label_2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>0</y>
        <width>31</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>H</string>
      </property>
     </widget>
     <widget class="QLabel" name="KRITICAL_Label_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>50</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>KRITICAL</string>
      </property>
     </widget>
     <widget class="QLabel" name="T_Label_2">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>100</y>
        <width>31</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>T</string>
      </property>
     </widget>
     <widget class="QPushButton" name="logOut_Button">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>LOG OUT</string>
      </property>
     </widget>
     <widget class="QPushButton" name="newTemplate_Button">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>200</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string>New Template</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loadTemplate_Button">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>200</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string>Load Template</string>
      </property>
     </widget>
     <widget class="QPushButton" name="testMode_Button">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>200</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string>Test Mode</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
